diff --git a/src/tomlutils.rs b/src/tomlutils.rs
index da793aa..e03fbac 100644
--- a/src/tomlutils.rs
+++ b/src/tomlutils.rs
@@ -9,7 +9,7 @@ use std::fs::read;
 use std::path::Path;
 
 use serde::de::DeserializeOwned;
-use tracing::{info, warn};
+use tracing::{debug, info, warn};
 
 /// Contains possible errors when loading/saving TOML from/to disk
 #[derive(thiserror::Error, Debug)]
@@ -28,9 +28,9 @@ pub type TomlFileResult<T> = Result<T, TomlFileError>;
 
 /// Load the TOML file from disk without any checks.
 fn load_raw_toml<T: DeserializeOwned>(path: &Path) -> TomlFileResult<T> {
-    Ok(toml::from_str(std::str::from_utf8(
-        read(path)?.as_slice(),
-    )?)?)
+    let bytes = read(path)?;
+    debug!("File contents: {:?}", bytes);
+    Ok(toml::from_str(std::str::from_utf8(bytes.as_slice())?)?)
 }
 
 /// Load the TOML file from disk.
